shared/js/ui/models/top-blocked.es6.js

```diff
@@ -20,17 +20,17 @@ TopBlocked.prototype = window.$.extend({},
       return new Promise((resolve, reject) => {
         this.fetch({getTopBlockedByPages: this.numCompanies})
           .then((data) => {
+            // TODO remove this before merging - added for testing
+            this.addTestData()
+            return resolve()
+
             if (!data.totalPages || data.totalPages < 30) return resolve()
             if (!data.topBlocked || data.topBlocked.length < 1) return resolve()
             this.companyList = data.topBlocked
             this.companyListMap = this.companyList.map((company) => {
               return {
                 name: company.name,
-                percent: company.percent,
-                // calc graph bars using pixels instead of % to
-                // make margins easier
-                // max width: 145px
-                px: Math.floor(company.percent / 100 * 145)
```

Comment:
On different OSes you can get scrollbars with differing widths, which means it's a bad idea to assume a fixed width for the external bar.

The internal bar really doesn't need a fixed width anyway. If its width is defined as a % of the outer bar, it'll be accurate.